Expensify/react-native-web
# Fix: [Violation] Added non-passive event listener to a scroll-blocking 'wheel' event

### Details


### Fixed Issues

$ https://github.com/Expensify/App/issues/26990
PROPOSAL: https://github.com/Expensify/App/issues/26990#issuecomment-1714914733


### Tests


1. Open Expensify ND
2. Verify that the warning: `[Violation] Added non-passive event listener to a scroll-blocking 'wheel' event` does not appear in the JS console


### Screenshots/Videos
<details>
<summary>Web</summary>



https://github.com/Expensify/react-native-web/assets/113963320/b78eba08-b679-40b5-bc5a-be074e408e59


</details>
